happy_bank_core/app.py

```diff
@@ -69,6 +84,24 @@ def health():
     return "Happy Bank Core app is up and running.", 200
 
 
+@api.route("/registration", methods=["POST"])
+def registration():
+    """Creates new account"""
+    content_type = request.headers.get("Content-Type")
+    if content_type == "application/json":
+        json_data = request.json
+        account = Registration.register(json_data)
+        file_connector = FileConnector()
+        file_connector.update(account)
+        return (
+            "You have successfully registered",
+            200,
+            {"Content-Type": "application/json"},
+        )
+    else:
+        return "Content-Type not supported!", 400
```

Comment:
In case, if block ends with return or raise statement, you can omit the else block by the pylint.

```suggestion
    return "Content-Type not supported!", 400
```